### PR TITLE
Removed inline style in HTML and added it to CSS #23

### LINKED ISSUE #23: remove inline style and add it to css
**Describe the code quality issue.**
there are some tags in html using inline styling which is not preferred 

**Expected Improvements**
remove them, and add them to css file of respective html file,

### PR DESCRIPTION
**Pull request by Akshat1276**
### Related Issue  
Closes #23 

### Type of Change
- [ ] New Feature  
- [ ] Bug Fix  
- [X] Code Refactor  
- [ ] Documentation Update  
- [ ] Other (please specify): 

### Description of Change  
Removed inline style in HTML found in `<div class="abc">` and `<div class="ijk">` and added it to CSS

###Demo
>before
image
image
image
image

>after
image
image
image
image

### CODE REVIEW

**Comment by Hardik-2311** [comment]
@Akshat1276 Could you please share a screenshot of your screen after making the changes, demonstrating the desired behavior?

**Comment by Akshat1276** [comment]
Shared the screenshots sir.

File: index.html [hunk 1]
```diff
@@ -139,13 +139,12 @@
...
-                            style="color: #F6FFFA;"></i></a></div>
+                            ></i></a></div>
                 <div class="def">Skibiddi toilet, Area 51</div>
                 <div class="abc"><a href="#"><i class="fa fa-2x fa-phone" class="contact"
-                            style="color: #F6FFFA;"></i></a></div>
+                            ></i></a></div>
                 <div class="def"> +91 4206969420</div>
-                <div class="abc"><a href="#"><i class="fa fa-2x fa-envelope" class="contact"
-                            style="color: #F6FFFA;"></i></a></div>
+                <div class="abc"><a href="#"><i class="fa fa-2x fa-en></i></a></div>
```
Comment (Yash-g2310) [inline index.html:147]: hey @Akshat1276  there are few issues with your commits
firstly there is a missing ' " ' here and the class name is also not set properly (match with the actual code)

**Review by Yash-g2310 — CHANGES_REQUESTED** [review]
@Akshat1276  i have mentioned the change required 
image
as you can see without it it is creating a bit of a mess
please change this asap
if any doubt head to [easysell discord](https://discord.gg/aKaEbaVYKf)

**Comment by Yash-g2310** [comment]
solved by someone else, thanks for contributing !

**Comment by Yash-g2310** [comment]
solved by someone else, thanks for contributing !

**Pull request closed without merging**